cpp/src/parquet/encryption/file_key_wrapper.cc

```diff
@@ -113,14 +113,15 @@ KeyEncryptionKey FileKeyWrapper::CreateKeyEncryptionKey(
     const std::string& master_key_id) {
   std::string kek_bytes(kKeyEncryptionKeyLength, '\0');
   RandBytes(reinterpret_cast<uint8_t*>(kek_bytes.data()), kKeyEncryptionKeyLength);
+  SecureString secure_kek_bytes(std::move(kek_bytes));
```

Comment:
Here as well, we could write directly into a pre-sized SecureString.

Reply:
done